code/modules/mining/machine_stacking.dm

```diff
@@ -81,24 +88,31 @@
 	var/stack_amt = 50 //amount to stack before releassing
 	var/datum/component/remote_materials/materials
 	var/force_connect = FALSE
+	var/io = INPUT //This is used for determining whether we change the Input or Output
+
+
+/obj/machinery/mineral/stacking_machine/examine(mob/user)
+	. = ..()
+		. += {"<span class='notice'>Currently outputting stacks at <b>[stack_amt] sheet[(stack_amt > 1) ? "s" : ""]</b><span>"}
+	if(panel_open)
+		. += {"The I/O is set to change [io ? "output" : "input"] currently.
+Input is <b>[dir2text(input_dir)]</b>
+Output is <b>[dir2text(output_dir)]</b>
+There is a dial that can be <b>turned by hand</b> to change direction.
+The I/O setting can be <b>changed with a multitool</b> to change what the dial controls
+There are some <b>bolts</b> to limit stack size."}
```

Comment:
```suggestion
There are some <b>bolts</b> to limit stack size."
```

Reply:
If I apply all of these together it just looks like a word salad to be honest.
![image](https://user-images.githubusercontent.com/57756830/71833294-3ca3e380-3072-11ea-9500-6c60a584a762.png)
But if this is how to keep it consistent with other machines, that would probably be best.

Reply:
In that case, for each line do . += "message here"

Reply:
![image](https://user-images.githubusercontent.com/57756830/71917609-164f7800-3146-11ea-9eca-78405f1c48a4.png)
This is what it looks like now.
Let me know if this needs to be changed at all.